pkg/rke/templates/templates.go

```diff
@@ -245,8 +246,10 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.13.0-rancher0":        canalv18,
 		},
 		kdm.Flannel: {
-			">=1.32.0-rancher1-1":                     flannelv0264,
-			">=1.29.0-rancher1-1 <1.32.0-rancher1-1":  flannelv0251,
+			">=1.32.12-rancher1-1":                    flannelv0273,
+			">=1.32.0-rancher1-1 <1.32.12-rancher1-1": flannelv0264,
+			">=1.32.11-rancher1-1 <1.32.0-rancher1-1": flannelv0273,
+			">=1.29.0-rancher1-1 <1.31.12-rancher1-1": flannelv0251,
```

Comment:
Ranges look off. 2 issues:
1. `v1.32.8-rancher1-1` is the version we've bumped Flannel on, the range is `>=1.32.12-rancher1-1` though
2. `>=1.32.11-rancher1-1 <1.32.0-rancher1-1` is certainly wrong, no versions fit it (looks like `1`/`2` were swapped accidentally.

Suggested fix, please double-check.

```suggestion
			">=1.32.8-rancher1-1":                     flannelv0273,
			">=1.32.0-rancher1-1 <1.32.8-rancher1-1":  flannelv0264,
			">=1.31.12-rancher1-1 <1.32.0-rancher1-1": flannelv0273,
			">=1.29.0-rancher1-1 <1.31.12-rancher1-1": flannelv0251,
```

Reply:
fixed.

Reply:
@krunalhinguu , maybe I'm missing something but the only change in updated ranges is `>=1.32.11-rancher1-1 <1.32.0-rancher1-1` -> `">=1.32.12-rancher1-1 <1.32.0-rancher1-1"` which is not correct or addresses both issues above.
Is there anything wrong with my suggested change in the previous comment?

Reply:
my bad, sorry for the confusion @snasovich